web/shapeworks/package.json

```diff
@@ -18,7 +18,7 @@
     "json-schema-defaults": "^0.4.0",
     "lodash": "^4.17.21",
     "shader-loader": "^1.3.1",
-    "vtk.js": "*",
+    "vtk.js": "^28.11.0",
```

Comment:
I suppose we don't, so I can put this back. I had been following the general axiom that pinning to major versions is a good idea because major versions can often come with breaking API changes. But in the case of vtk.js, this might hold us back. What about specifying `>=28.11.0` so we can be explicit about a minimum version but still allow future major versions?

Reply:
Changed in [7cb20c6](https://github.com/girder/shapeworks-cloud/pull/340/commits/7cb20c656b81a0bd6cc96c1b5dc63bf6dbbc53e1)